### ISSUE TITLE
Make the error generated by using tuple indexing on an array clearer.

### ISSUE THREAD

**Issue by Tom1380** [rust-lang/rust#53713]
Compiling this:
```rust
let a = [100, 100];
a.0
```
produces this error:
```
error[E0609]: no field `0` on type `[{integer}; 2]`
```
I think that it should be changed to something along the lines of:
```
error[E----]: attempting to use tuple indexing on an array.
```
and give this hint:
```
help: change this to `a[0]`
```

**Comment by varkor** [comment]
I think you accidentally posted two of these issues! Closing in favour of https://github.com/rust-lang/rust/issues/53712.

**Comment by Tom1380** [comment]
@varkor Yes, I accidentally closed the browser tab I was writing #53712 in and I thought I had to start over. Instead, it was posted. Thanks, also, it's nice to see Rust community's being so active!
